parameters/reference/binarization-modes.md

```diff
@@ -8,9 +8,9 @@ description: Dynamsoft Document Normalizer Parameter Reference - BinarizationMod
 
 # BinarizationModes
 
-This parameter helps control the process of binarization, i.e., converting grayscale image to binary image. During the binarization, as we know, the threshold is the key criteria. If the pixel value is smaller than the threshold, it is set to 0, otherwise, it is set to a maximum value (255 in the library). By default, the library automatically calculates the adaptive size of the neighbourhood area and then binarizes the grayscale image with the adaptive threshold based on a small neighbourhood area with an adaptive size around it. Assume this field is in your wheelhouse, this parameter may help specify the appropriate settings for getting a better binary image.
+This parameter helps control the binarization process, which converts a grayscale image to a binary image, and potentially result in a better quality binary image. During binarization, the threshold is the key criteria. If the pixel value is smaller than the threshold, it is set to 0. Otherwise, the threshold is set to the maximum value (255 in the library). By default, the library automatically calculates the adaptive size of the neighbourhood area and then binarizes the grayscale image with the adaptive threshold based on a small neighbourhood area with an adaptive size around it.
```

Comment:
It's the pixel value on the grayscale which will be set to 0 or 255 based on the threshold, not the threshold itself